### ISSUE TITLE
The description is required for profiles

### ISSUE THREAD

**Issue by alficles** [apache/trafficcontrol#2502]
Users should be reasonably able to leave this field blank, but it is marked as required.

**Comment by mitchell852** [comment]
@alficles - what are we talking about here? TP?

**Comment by alficles** [comment]
Apparently, we're talking about not enough caffeine before writing bug tickets, cause you're right, that's totally not enough info. :)

In Traffic Portal, the Description field is required for profiles. (I think it may also be marked unique, which is somewhat confusing.) In general, it's desirable for profiles to have descriptions, but there's not solid technical reason users should be required to put something in the box.

Users already tend to put quite a bit of "information" in the name of the profile itself. In the wild, I've observed a tendency for the description to just be an unpacked version of the information in the name. That's fine, of course, but users should be free to choose whether they want to put something there.

If we don't fix this, then we should probably fix the fact that when you copy a profile, the description is cleared, but the copy is successful. This means you don't need a description until you go to edit the profile for the second time.

**Comment by mitchell852** [comment]
fyi, the `description` field is currently required in the API. Doesn't mean that constraint can't be removed from the API but I'm just saying.

**Comment by mitchell852** [comment]
> If we don't fix this, then we should probably fix the fact that when you copy a profile, the description is cleared, but the copy is successful. This means you don't need a description until you go to edit the profile for the second time.

@alficles - not sure when this was fixed but cloning a profile does preserve the description now. want to close this issue?

**Comment by alficles** [comment]
I don't think so. I don't see a reason the description needs to be mandatory. We can take the "bug" tag off, though, because this is just a change in functional behaviour.

**Comment by mitchell852** [comment]
i agree. it probably should have never been required in the first place.

**Comment by mitchell852** [comment]
i added the label used to tag things for the version of the api where we plan to make many changes to the api
